eth/downloader/downloader.go

```diff
@@ -209,10 +209,16 @@ type BlockChain interface {
 
 	// InsertReceiptChain inserts a batch of receipts into the local chain.
 	InsertReceiptChain(types.Blocks, []types.Receipts, uint64) (int, error)
+
+	// Config returns the block chain configuration
+	Config() *params.ChainConfig
```

Comment:
I didn't see any usage of Config()

Reply:
Yes, had added it when I wanted to check from config if it is Quorum. Code changes done.